addons/mail/static/src/components/chat_window_header/chat_window_header.js

```diff
@@ -50,6 +50,18 @@ class ChatWindowHeader extends Component {
         this.trigger('o-clicked', { chatWindow });
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickChatHeaderName(ev) {
+        if (ev.currentTarget.classList.contains('o_Clickable_ChatWindowHeaderName')) {
+            this.chatWindow.thread.correspondent.openProfile();
+        } else {
+            this._onClick();
```

Comment:
Don't call this other onclick manually, let the event buble instead.

See `markEventHandled` and `isEventHandled`.